app/ts-cli/geminiql/lexer.go

```diff
@@ -133,6 +133,14 @@ func (t *Tokenizer) scanIdentifier() (int, string) {
 		ch := t.read()
 		if ch == EOF {
 			break
+		} else if ch >= 128 {
```

Comment:
This way of writing supports `emoji` is inappropriate.

Reply:
I have completed the modifications according to the above plan.